website/source/docs/platform/k8s/helm.html.md

```diff
@@ -377,6 +377,176 @@ to run the sync program.
+      `targetPort` will automatically be set to `meshGateway.containerPort`.
+
+      * <a name="v-meshgateway-service-nodeport" href="#v-meshgateway-service-nodeport">`nodePort`</a> (`int`) -
+      Optional [`nodePort`](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) of the `Service`. Can be used in conjunction with
+      `type: NodePort`.
+
+      * <a name="v-meshgateway-service-annotations" href="#v-meshgateway-service-annotations">`annotations`</a> (`string`) -
+      Optional YAML string for additional annotations on the Service. This should be a formatted as a multi-line string.
+
+        ```yaml
+        annotations: |
+          "sample/annotation1": "foo"
+          "sample/annotation2": "bar"
+        ```
+
+      * <a name="v-meshgateway-service-additionalspec" href="#v-meshgateway-service-additionalspec">`additionalSpec`</a> (`string`) -
+      Optional string for appending additional YAML to the Service spec. This should be a formatted as a multi-line string.
+
+        ```yaml
+        additionalSpec: |
+          clusterIP: 10.0.171.239
+        ```
+
+  * <a name="v-meshgateway-imageenvoy" href="#v-meshgateway-imageenvoy">`imageEnvoy`</a> (`string: "envoyproxy/envoy:v1.10.0"`) -
+  [Envoy](https://www.envoyproxy.io/) Docker image to use. Envoy is the proxy used for the gateway.
+
+  * <a name="v-meshgateway-hostnetwork" href="#v-meshgateway-hostnetwork">`hostNetwork`</a> (`bool: false`) -
+  If set to true, gateway Pods will run on the host network.
+
+  * <a name="v-meshgateway-dnspolicy" href="#v-meshgateway-dnspolicy">`dnsPolicy`</a> (`string`) -
+  If set, this will be the dns policy used. If `hostNetwork` is `true` then this should be set to `ClusterFirstWithHostNet`.
+
+  * <a name="v-meshgateway-consulservicename" href="#v-meshgateway-consulservicename">`consulServiceName`</a> (`string`) -
+  Override the default 'mesh-gateway' service name registered in Consul.
+  Cannot be used if `bootstrapACLs` is true since the ACL token generated
+  is only for the name 'mesh-gateway'.
+
+  * <a name="v-meshgateway-containerport" href="#v-meshgateway-containerport">`containerPort`</a> (`int: 443`) -
+  Port that the gateway will run on inside the container.
+
+  * <a name="v-meshgateway-hostport" href="#v-meshgateway-hostport">`hostPort`</a> (`int`) -
+  Optional host port for the gateway to be exposed on.
+  This can be used with `wanAddress.port` and `wanAddress.useNodeIP`
+  to expose gateways directly on the node.
+  If hostNetwork is true, this must be null or set to the same port as
+  containerPort.
+
+  * <a name="v-meshgateway-enablehealthchecks" href="#v-meshgateway-enablehealthchecks">`enableHealthChecks`</a> (`bool: true`) -
+  If there are no connect-enabled services running, then the gateway
+  will fail health checks. You may disable health checks as a temporary
+  workaround.
+
+  * <a name="v-meshgateway-resources" href="#v-meshgateway-resources">`resources`</a> (`string`) -
```

Comment:
All the code blocks under bullet points on this page look like they might need to be indented one more level to nest correctly under the bullets.

Reply:
👍